hack/verify-upgrade.sh

```diff
@@ -22,16 +22,22 @@ SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" > /dev/null && pwd )"
 export REPO_ROOT="${SCRIPT_ROOT}/.."
 source "${REPO_ROOT}/hack/build/version.sh"
 
-kube::version::last_published_release
-
-LATEST_RELEASE="${KUBE_LAST_RELEASE}"
+# This script is copied from k8s and modified.
+# It helps to determine the latest published release with the given prefix
+# (if INITIAL_RELEASE is not explicitly set) and also provides the latest git
+# commit hash
+kube::version::last_published_release ${INITIAL_RELEASE_PREFIX:-"v*"}
+
+if [[ -z "${INITIAL_RELEASE:-}" ]]; then
+	INITIAL_RELEASE="${KUBE_LAST_RELEASE}"
+fi
 
 usage_and_exit() {
-	echo "usage: $0 <path-to-helm> <path-to-kind> <path-to-ytt> <path-to-kubectl> <path-to-cmctl>" >&2
+	echo "usage: $0 <path-to-helm> <path-to-kind> <path-to-ytt> <path-to-kubectl> <path-to-cmctl> <host-architecture>" >&2
```

Comment:
note: The upgrade test was broken on non-amd64 architectures before this change!